src/main/scala/io/iohk/ethereum/rlp/package.scala

```diff
@@ -0,0 +1,36 @@
+package io.iohk.ethereum
+
+package object rlp {
+
+  case class RLPException(message: String) extends RuntimeException(message)
+
+  sealed trait RLPEncodeable
+
+  case class RLPList(items: RLPEncodeable*) extends RLPEncodeable
+
+  case class RLPValue(bytes: Array[Byte]) extends RLPEncodeable
+
+  trait RLPEncoder[T] {
+    def encode(obj: T): RLPEncodeable
+  }
+
+  trait RLPDecoder[T] {
+    def decode(rlp: RLPEncodeable): T
+  }
+
+  def encode[T](input: T)(implicit enc: RLPEncoder[T]): Array[Byte] = RLP.encode(enc.encode(input))
```

Comment:
@lukasGasior1, in this case the API for RLP is small enough to be declared in the package class. That helps clients of the module understand what's available at a glance. When clients import the package they have all the functionality available bar the supporting implicits.